gtsam/slam/BoundingConstraint.h

```diff
@@ -133,15 +138,27 @@ struct BoundingConstraint2: public NoiseModelFactorN<VALUE1, VALUE2> {
       OptionalMatrixType H1 = OptionalNone,
       OptionalMatrixType H2 = OptionalNone) const = 0;
 
-  /** active when constraint *NOT* met */
-  bool active(const Values& c) const override {
-    // note: still active at equality to avoid zigzagging
-    double x = value(c.at<X1>(this->key1()), c.at<X2>(this->key2()));
-    return (isGreaterThan_) ? x <= threshold_ : x >= threshold_;
+  Vector unwhitenedExpr(const Values& x, OptionalMatrixVecType H = nullptr) const override {
```

Comment:
same question about active
what is `unwhitenedExpr`? Not documented

Reply:
see above. `unwhitenedExpr` is documented in the base class.